data/strings/strings.txt

```diff
@@ -4465,6 +4465,11 @@
     zh-Hans = 语音指导
     zh-Hant = 語音指示
 
+  [pref_tts_street_names_title]
+    comment = Settings «Route» category: «Tts announce street names» title
+    tags = android,ios
+    en = Announce Street Names
+
```

Comment:
pl = Oznajmianie nazw ulic

Reply:
I think we should find a better translation because it's not only about street names but for destination and road numbers.

Reply:
The English version also doesn't specify those things, so it's more of an issue with the original text than the translation

Reply:
A future improvement may have separate logic for exits so that we can say "turn right onto 11th Avenue" and then separately "take exit 123 onto 11th Avenue for City Center towards London" but that's a high level of parsing and complexity I'd like to avoid for now if at all possible.

Reply:
> but that's a high level of parsing and complexity I'd like to avoid for now if at all possible.

It would be great to document all necessary cases to see the whole picture. And maybe put a good foundation to properly support these changes in the future. Our vision is that everything should eventually be perfect, for all countries, and for all languages.

Reply:
@lunarna-gh what do you think the best way to communicate announcing both street names and exits would be?

Reply:
I'm not sure, I don't think it's that important, but if needed there could just be smaller text underneath that states "When turned on, the text-to-speech voice will announce street names, exit signs and reference codes." (if it says reference codes). This could also make the option easier to understand in some languages while avoiding making the setting title too long

Reply:
> I'm not sure, I don't think it's that important, but if needed there could just be smaller text underneath that states "When turned on, the text-to-speech voice will announce street names, exit signs and reference codes." (if it announces reference codes). This could also make the option easier to understand in some languages while avoiding making the setting title too long

That's what I've been thinking of. With that the title itself is good and we just need to add explanation as you said. It's probably the best way to go ATM. :)

Reply:
@lunarna-gh @pm4rcin please see https://zyphlar.github.io/organicmaps-locale-viewer/ and the updated source files, I've changed what I believe is your consensus so please let me know if it's better or worse.